distributed/tests/test_client.py

```diff
@@ -2195,6 +2195,23 @@ def test_run(c, s, a, b):
     assert results == {}
 
 
+@gen_cluster(client=True)
+def test_run_serialize(c, s, a, b):
```

Comment:
I've expanded the title to be more informative and changed the code to be, I think, more self explanatory.  There was no related github issue.  The problem was reported in private conversation.